### PR TITLE
[Functionality] Building Submit Test Footer

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

I created a new generic component called '_TestFooter.jsx_', so we can add this footer in all the desired tests.

The **Submit Test** button still works as expected, but the **Quit Test** button doesn't have any action/behavior for now. I'll create another PR with a new generic popup window component and add an action to this **Quit Test** button.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Instruction Tab [EN]:**

image

**Instruction Tab [FR]:**

image

**Background Tab [EN]:**

image

**Background Tab [FR]:**

image

**Inbox Tab [EN]:**

image

**Inbox Tab [FR]:**

image

# Testing

Manual steps to reproduce this functionality:

1.  Go to _Prototype_ tab.
2.  Click _Start eMIB Sample Test_.
3.  Navigate in the test and make sure that the **Submit Test** button still works as expected.
4. Navigate in all browser to make sure everything looks great as well.

Screenshot of unit tests run passing:

image

# Checklist

Applicable for all code changes.

- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] My changes generate no new compiler warnings
- [ ] ~~My changes generate no new accessibility errors and/or warnings~~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 10+, Firefox, and Chrome

### CODE REVIEW

**Comment by MichaelCherry** [comment]
Just noting that the inbox tab looks a little funky when the screen is really short. However, the test would be almost impossible to take if viewed like this and we will be replacing Inbox anyway, so I'd say this is a non-issue:

image

File: frontend/src/components/eMIB/EmibTabs.jsx [hunk 1]
```diff
@@ -10,10 +9,6 @@ import Notepad from "../commons/Notepad";
 import "../../css/emib-tabs.css";
 
 class EmibTabs extends Component {
-  static propTypes = {
```
Comment (caleybrock) [inline frontend/src/components/eMIB/EmibTabs.jsx]: don't you also need to remove this from render?

**Comment by fnormand01** [comment]
I just fixed the merge conflicts. Here is a last tests run:

image

File: frontend/src/components/eMIB/EmibTabs.jsx (hunk 1 above)
Comment (fnormand01) [reply frontend/src/components/eMIB/EmibTabs.jsx]: Fixed in merge conflicts commit #https://github.com/code-for-canada/project-thundercat/pull/122/commits/86955f686dc53e749f4b41244a89bb852afdbabd

**Pull request merged**